docs/guides/migrating-cra.md

```diff
@@ -0,0 +1,774 @@
+
+Most apps bootstrapped with [Create React App](https://create-react-app.dev/) (CRA) already use [React Router](https://reactrouter.com/) for client-side routing. Because Remix is built on top of React Router, migration should be a relatively easy process.
+
+If you aren't already using React Router, we think there are several compelling reasons to reconsider! History management, dynamic path matching, nested routing, and much more. Take a look at the [React Router docs](https://reactrouter.com/docs/en/v6/getting-started/concepts) and see what all we have to offer.
+
+## Ensure your app uses React Router v6
+
+If you are using an older version of React Router, the first step is to upgrade to v6. Check out the [migration guide from v5 to v6](https://reactrouter.com/docs/en/v6/upgrading/v5) and our [backwards compatibility package](https://www.npmjs.com/package/react-router-dom-v5-compat) to upgrade your app quickly.
+
+## A note about non-standard imports
+
+CRA supports several non-standard file imports in your JavaScript/TypeScript modules. Since such imports don't actually work in plain JavaScript, they have to be transformed by a compiler into something that makes sense at runtime. This includes things like styles, images, and SVG files.
+
+Remix also supports some non-standard imports, but it's not a 1-to-1 mapping in terms of what we support or how we handle loading the file contents. Below is a non-exhaustive list of some of the differences you'll encounter in Remix.
+
+### Images and fonts
+
+CRA allows you to import image files, and result is a string representing the filepath of the image. This works the same way in Remix.
+
+```js
+import * as React from "react";
+
+import logo from "./logo.png";
+
+export function Logo() {
+  return <img src={logo} alt="My logo" />;
+}
+```
+
+This should work for fonts as well. In Remix, you'll generally import fonts in a route module and use them as the `href` of a `link` tag. [See our route loader docs for more information.](../api/conventions#loader)
+
+### SVG
+
+CRA allows you to import SVG files as React component. This is a common use case for SVG files, but it's not supported by default in Remix.
+
+```js
+import * as React from "react";
+
+// This will not work in Remix!
+import { ReactComponent as MyLogo } from "./logo.svg";
+
+export function Logo() {
+  return <MyLogo />;
+}
+```
+
+If you want to use SVG files as React components, you'll need to first create the components and import them directly. [React SVGR](https://react-svgr.com/) is a great toolset that can help you generate these components from the [command line](https://react-svgr.com/docs/cli/) or in an [online playground](https://react-svgr.com/playground/) if you prefer to copy and paste.
+
+### Style imports
+
+CRA supports importing styles in your components in many ways. While this is common practice in the React ecosystem, it's not supported the same way in Remix for a few different reasons.
```

Comment:
I think this section could be expanded to list a few of the common issues people will come up against due to this and what they should do about it.

Reply:
Yeah I meant to include a link to the section below where we talk about styles in relation to `links` but I just missed it 😅